### DISCUSSION TITLE
Tuple length matching error with mfhfb.py

### DISCUSSION THREAD

**Discussion by CynicalDonkey** [modflowpy/flopy#1672] [Q&A]
I am working with flopy 3.3.7 and attempting to load a fresh copy of the Texas Water Development Board central carrizo wilcox groundwater availability model, which is modflow-USG, to Spyder. After running flopy.mfusg.MfUsg.load(), I get the following error:
```
  File C:\anaconda3\lib\site-packages\flopy\modflow\mfhfb.py:326 in load
    specified[ibnd] = tuple(t[: len(specified.dtype.names)])

ValueError: could not assign tuple of length 3 to structure with 6 fields.
```
I've opened the mfhfb.py file to see if I can determine what is happening, but it's a bit beyond my comprehension. The section where the error in mfhfb.py is thrown from is below:
```
        # data set 4
        bnd_output = None
        if nhfbnp > 0:
            specified = ModflowHfb.get_empty(nhfbnp)
            for ibnd in range(nhfbnp):
                line = f.readline()
                if "open/close" in line.lower():
                    raise NotImplementedError(
                        "load() method does not support 'open/close'"
                    )
                t = line.strip().split()
                specified[ibnd] = tuple(t[: len(specified.dtype.names)])
```
Any help is appreciated.

**Comment by cnicol-gwlogic** [comment]
Hi @CynicalDonkey I'll take a look at this in the next week or so.

**Comment by cnicol-gwlogic** [comment]
@CynicalDonkey see PR #1686. Once/if accepted, you can update your flopy install to the latest develop branch and then be able to load your model, eg:
```
pip uninstall --yes flopy
pip install https://github.com/modflowpy/flopy/archive/refs/heads/develop.zip
```

**Comment by CynicalDonkey** [comment]
Chris,

Thanks very much for taking the time to address this and push a fix. If you
have time at some point, I would love to have a general chat about the big
picture view of what you changed. I have taken a look through the code
commit, but it's a bit beyond my understanding at the moment. That is what
I am working on though, and getting feedback or assistance from people who
know their stuff, as it seems you do, always helps out a lot.

Thanks,

Wesley

**Reply by cnicol-gwlogic** [reply]
No worries at all. You can message me at: info /at/ groundwaterlogic.com
